runtime/util_core/j9argscan.c

```diff
@@ -111,8 +115,25 @@ void scan_failed(J9PortLibrary * portLibrary, const char* module, const char *sc
 uintptr_t scan_double(char **scan_start, double *result)
 {
 	char *endPtr = NULL;
-
+#if defined(WINDOWS)
+	_locale_t clocale = _create_locale(LC_ALL, "C");
+	*result = _strtod_l(*scan_start, &endPtr, clocale);
+	_free_locale(clocale);
+#elif defined(OSX) /* defined(WINDOWS) */
+	locale_t clocale = newlocale(LC_ALL_MASK, "C", NULL);
+	*result = strtod_l(*scan_start, &endPtr, clocale);
```

Comment:
`strtod_l()` appears to be a BSD extension on OSX, why not just use the Linux pattern on OSX if we're going to depend on BSD-isms?

Reply:
Sure, updated.